src/Surfnet/StepupRa/RaBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -115,13 +115,6 @@ public function authenticate(TokenInterface $token)
             }
         }
 
-        // Get authorizations (implicit RAA roles through select_raa institution configuration)
-        if (!empty($profile->implicitRaaAt)) {
-            // The ROLE_SELECT_RAA applies, regardless of which institution it applies for, the application will only
-            // allow RAA actions for those institutions.
-            $roles[] = 'ROLE_SELECT_RAA';
```

Comment:
It was added because we assumed that if a user was made RAA for an institution because that could because of SELECT_RAA we didn't take USE_RAA into account anymore to prevent a RAA from being locked out. This however was not a valid assumption and prevented the use 'virtual' organizations.